get-latest-pulsecore.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -ex
+
+exit_updater() {
+    local exit_code=$?
+    rm -rf "$TMPDIR"
+    rm -f "$LOCALDIR/${SRC_FILE:?}"
+    if [ ${exit_code} -ge 1 ]; then
+        echo "-> An error occurred while fetching latest pulsecore headers. Manual update is required"
+    fi
+    exit "${exit_code}"
+}
+
+LOCALDIR="$(readlink -f "$(dirname "$0")")"
+BUILDERDIR="$(readlink -f "$LOCALDIR/../builder-rpm")"
+TMPDIR="$(mktemp -d)"
+
+if [ ! -d "$BUILDERDIR" ]; then
+    echo "Cannot find qubes-builder-rpm. Exiting..."
+    exit 1
+fi
+
+LATEST_FEDORA_RELEASE="$(git ls-remote --heads https://src.fedoraproject.org/rpms/fedora-release | grep -Po "refs/heads/f[0-9][1-9]*" | sed 's#refs/heads/f##g' | sort -g | tail -1)"
+LATEST_FEDORA_VERREL="$(dnf -q repoquery pulseaudio --disablerepo=* --enablerepo=fedora --enablerepo=updates --releasever="$LATEST_FEDORA_RELEASE" | sort -V | tail -1 | cut -d':' -f2 | sed "s/.fc.*x86_64//")"
```

Comment:
please check for expected characters/format - for example with `grep` like you do above. This may cover the thing cut+sed currently does.

Reply:
I've improved it.